rs/crypto/internal/crypto_lib/threshold_sig/tecdsa/src/lib.rs

```diff
@@ -849,6 +849,9 @@ impl From<CanisterThresholdError> for ThresholdEcdsaCombineSigSharesInternalErro
 ///
 /// The signature shares must be verified prior to use, and there must
 /// be at least reconstruction_threshold many of them.
+///
+/// All shares must have been created with respect to the same derivation path,
+/// message, randomness, and transcripts.
```

Comment:
Should we have similar comments on the top level traits `ThresholdSEcdsaSigVerifier::combine_sig_shares` of the crypto component? In that context maybe we can just say that the precondition is that all the signature shares should be valid with respect to the given `ThresholdEcdsaSigInputs`. Same for Schnorr.

Reply:
Added comments there as well